Apps/phpassivetotal/passivetotal.json

```diff
@@ -9,7 +9,7 @@
     "product_vendor": "PassiveTotal",
     "product_name": "PassiveTotal",
```

Comment:
please update the publisher to Splunk (not this line, but github didn't let me comment on the relevant line)

Reply:
Done